shell/browser/ui/cocoa/electron_bundle_mover.mm

```diff
@@ -181,19 +181,27 @@
 bool ElectronBundleMover::IsCurrentAppInApplicationsFolder() {
   return IsInApplicationsFolder([[NSBundle mainBundle] bundlePath]);
 }
+  
+NSString* resolvePath(NSString* path) {
+  NSString *standardizedPath = [path stringByStandardizingPath];
+  char resolved[PATH_MAX];
+  if (realpath([standardizedPath UTF8String], resolved) == NULL) return path;
+  return [NSString stringWithCString:resolved encoding:NSUTF8StringEncoding];
```

Comment:
you can use `[NSString stringWithUTF8String:resolved]` or just @(resolved)`

Reply:
done, thx